packages/commonwealth/server/util/requirementsModule/requirementsSchema_v1.json

```diff
@@ -6,14 +6,18 @@
       "properties": {
         "source_type": {
           "type": "string",
-          "enum": ["erc20", "erc721"]
+          "enum": ["erc20", "erc721", "erc1155"]
         },
         "evm_chain_id": {
           "type": "number"
         },
         "contract_address": {
           "type": "string",
           "pattern": "^0x[a-fA-F0-9]{40}$"
+        },
+        "id": {
+          "type": "string",
+          "pattern": "^[0-9]+$"        
         }
```

Comment:
What's the `id` for?

FYI: the schema should only contain blockchain metadata, and should not reference anything from the DB.

Reply:
To query a balance of an erc1155 we need to provide the contract address and the id number of the token on chain, so assuming that is the purpose of id here

Reply:
In that case, can we make the field name more explicit, like `token_id`?

Reply:
Yes that is actually what TBC uses so that makes sense here as well @mw2000